distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Delimited_Writer.enso

```diff
@@ -95,10 +128,13 @@ write_to_writer table format java_writer =
     quote_characters = case format.quote_style of
         Quote_Style.No_Quotes -> Pair Nothing Nothing
         Quote_Style.With_Quotes _ quote quote_escape -> Pair quote quote_escape
-    write_headers = case format.headers of
-        True -> True
-        Infer -> True
-        False -> False
+    write_headers = should_write_headers format.headers
     new_line = '\n'
     writer = DelimitedWriter.new java_writer column_formatters.to_array format.delimiter new_line quote_characters.first quote_characters.second quote_behavior write_headers
     writer.write table.java_table
+
+## PRIVATE
+should_write_headers headers = case headers of
+    True -> True
+    Infer -> True
+    False -> False
```

Comment:
Yeah, well it can also be:
```
case headers of
    False -> False
    _ -> True
 ```

but I wanted to make it explicit which of the three options map to which. IMO the codes where the other two options are collapsed are slightly less readable - it is less clear which options map to which.